bip-0384.mediawiki

```diff
@@ -71,8 +71,8 @@ Invalid descriptors
 ==Backwards Compatibility==
 
 <tt>combo()</tt> descriptors use the format and general operation specified in [[bip-0380.mediawiki|380]].
-As this is a wholly new descriptor, it is not compatible with any implementation.
-However the scripts produced are standard scripts so existing software are likely to be familiar with them.
+As this is a wholly new descriptor, it is not compatible with any implementations.
+However the scripts produced are standard scripts so existing software is likely to be familiar with them.
```

Comment:
are is correct, software is plural

Reply:
https://github.com/bitcoin/bips/pull/1720

@achow101 
done

Reply:
There is no need to close a PR and open a new one to make a change to it.